frontend-next-backup/.gitignore

```diff
@@ -0,0 +1,94 @@
+# Custom
```

Comment:
Go ahead and delete the entire `frontend-next-backup` directory.

We will still have a backup of the old history by being able to go back and look at the history in the git repo.